### PR TITLE
Add python 3.10 on ppc64le

### PR DESCRIPTION
**Pull request by skupr-anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,8 +37,9 @@ outputs:
         - certifi >=2020.06.20
         - freetype >=2.3
         - numpy   1.19  # [not (osx and arm64) and (py==37 or py==38 or py==39)]
+        - numpy   1.19  # [(linux and ppc64le) and (py==37 or py==38 or py==39)]
```
Comment (adipietro18) [inline recipe/meta.yaml:40]: Out of curiosity, is this covered by the line above? Can they be combined?

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:40]: I tried to combine them but readability is getting worse

Comment (skupr-anaconda) [reply recipe/meta.yaml]: @adipietro18 Oh, finally, I fixed it

**Pull request merged**